### PR TITLE
Fix home register form when using HTTPS

### PR DESCRIPTION
**Pull request by duhow**
Javascript forces to use HTTP. On HTTPS-only sites this breaks usability.
By using `//` the browser will keep using the current protocol.

### CODE REVIEW

**Comment by LucasKovacs** [comment]
@duhow please request merges to development

**Comment by duhow** [comment]
@LucasKovacs done :)

File: src/upload/application/views/home/index_body.php [hunk 1]
```diff
@@ -194,14 +194,8 @@
...
         </tbody></table>
-        <input class="js_uniUrl" type="hidden" name="uni_url" id="uni_domain" value="">
+        <input class="js_uniUrl" type="hidden" name="uni_url" id="uni_domain" value="{base_path}">
         <div id="uni_selection" style="display: none;">
-                                    <script type="text/javascript">
-                        <!--
-                            select_uni('{base_path}'.replace('http://', '').replace('https://', '').slice(0, -1), '','');
-                        //-->
-                        </script>
-
```
Comment (LucasKovacs) [inline src/upload/application/views/home/index_body.php:204]: @duhow this should not be part of this PR, it is unrelated

**Review by LucasKovacs — CHANGES_REQUESTED** [review]

File: src/upload/application/views/home/index_body.php (hunk 1 above)
Comment (duhow) [reply src/upload/application/views/home/index_body.php:204]: No, this is required. If "universe" is not set or empty, the JS function will **POST** to another domain.
Also as the Javascript code uses the full URL and not the `base_path`, having this code just breaks the domain.
So while we don't allow multi-universe, this code can be removed.

**Comment by duhow** [comment]
@LucasKovacs your change def20af2b214ce35cd2c1ef9ba1592fc30fd6d00 conflicts with the change i suggested in `index_body.php` , i'll just leave the commit for the JS file
